civis/tests/test_io.py

```diff
@@ -144,42 +149,396 @@ def test_large_file_to_civis(self, *mocks):
 
         assert isinstance(result, int)
 
+    @pytest.mark.civis_to_file
     @mock.patch(api_import_str, return_value=civis_api_spec)
     def test_civis_to_file(self, *mocks):
         buf = BytesIO()
         civis.io.civis_to_file(self.file_id, buf)
         buf.seek(0)
         assert buf.read() == b'a,b,c\n1,2,3'
 
-    @mock.patch(api_import_str, return_value=civis_api_spec)
```

Comment:
I know you were having issues with the cassettes and I totally support removing them. But is there any way to do a manual check that the old tests still pass with the new code? Since we're changing the code and the tests at the same time, I'd like to check that we're not breaking the API.

Reply:
I'm not sure exactly what you mean by "that the old tests still pass" ? They would fail, and be right to do so. The API calls that the cassettes were recording would no longer be hit, and VCR would complain about unexpected calls being made. I am happy to run live tests against Platform with the new code if that would help put worries to rest?

Reply:
Ah, you're completely right. I'm basically trying to think of a way to check that existing code that calls the convenience functions won't break, and that the user will get fundamentally the same behavior. But the old tests won't do that. If you could run a couple live tests with the new code, that would be helpful. Mostly because it's a fairly large/complex PR, so it's very possible for me to miss something.

Reply:
In reviewing this code, I actually realized that we _had_ introduced a new issue: in changing from directly constructing the`CivisFuture` to using `run_job`, `polling_interval` seemed to get incorrectly passed. This was because `run_job` didn't take a `polling_interval` argument. I have added an optional one, and used in the appropriate methods.

Other than that, I have created a [platform notebook](https://platform.civisanalytics.com/spa/#/notebooks/13325) to run the convenience methods. I replicated this notebook locally using the new implementation, (I would have loved to do so in platform, but ran into some issues with installing `civis` from my fork in a notebook) and verified that the results are the same as they were under the old implementation.

Please let me know if there are other tests besides these that you would like to see run!